index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+      .container{
+  display: flex;
+}
+    </style>
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <h1>Hello World, we are software developers! </h1>
+    <script  src="js/index.js"></script>
+    <header id="header">
+      <img src="http://img.jpg" alt="texlogo" id="header-img">
```

Comment:
- [ ] A better approach to state the issue could be **"This Image is not showing up in the browser I think it's due to broken image link 🤔 so please consider adding a correct image URL 👍🏼**

- Adding emojis will help your comment to be delivered in the right tone and also it will sound more human friendly 💪🏼